### ISSUE TITLE
Can I get your pre-trained model on the voc dataset ??

### ISSUE THREAD

**Issue by llmettyll** [Zehaos/pycaffe-yolo#2]
With VOC dataset, I failed training several times although I used same python codes that you've posted. 
I tried changing solver type, such as "Nesterov", "adam" instead of "SGD".
But I cannot succeed.
In my case, batch size was limited to '16' (on account of memory shortage of GPU, gtx1060).

After 200000 iteration, 
The 'loss' was over 50.
& 'Avg IOU' was under 0.4
& 'Pos Obj' was under 0.1

I think that it's totally wrong.
I want to check each variable value line by line precisely based on well-trained caffemodel.

Would you provide me a pre-trained model on VOC dataset?
